### PR TITLE
Admin settings 156 merge

### PR DESCRIPTION
**Pull request by wingfeatherwave**
screencapture-localhost-3000-circles-25-admin-2022-08-08-17_31_24

* Merging in @sshmm 1.5.6 work
* Cleanup admin settings form page
* @crabsinger there was a problem, with avatar, please confirm that my merge works properly with your efforts

### CODE REVIEW

**Review by sshmm — APPROVED** [review]
Thanks for the merge and added styles
I tested the functionality and everything work as before.

File: src/ui/TextField/TextField.tsx [hunk 1]
```diff
@@ -35,13 +35,14 @@ export const TextField = styled('input', {
         fontSize: '$small',
       },
       md: {
-        width: '250px',
-        height: '48px',
+        // width: '250px',
+        // height: '48px',
```
Comment (crabsinger) [inline src/ui/TextField/TextField.tsx:39]: @wingfeatherwave ?

**Review by crabsinger — CHANGES_REQUESTED** [review]
@wingfeatherwave looks good - one remianing fixup:

The DeWork delete button seems to render weirdly:

image

File: src/ui/TextField/TextField.tsx (hunk 1 above)
Comment (crabsinger) [reply src/ui/TextField/TextField.tsx:39]: is this intentinoal or can be removed?

Comment (wingfeatherwave) [reply src/ui/TextField/TextField.tsx:39]: Yes, I will remove.  And will resolve the dework button issue.  ty!

**Comment by wingfeatherwave** [comment]
Why does this show 1 change requested?  I thought that I had resolved it.  Perhaps I'm not doing something correctly in the resolution process?

**Review by crabsinger — APPROVED** [review]
This looks good to me.

**Pull request merged**